flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/LocalExecutorITCase.java

```diff
@@ -144,8 +150,9 @@ public void testCompleteStatement() {
         executor.closeSession(sessionId);
     }
 
-    @Test(timeout = 90_000L)
-    public void testStreamQueryExecutionChangelog() throws Exception {
+    @Test
+    @Timeout(90)
```

Comment:
Since JUnit 5 supports specification of timeunits it would be much more convenient to use them like 
```java
@Timeout(value = 90, unit = TimeUnit.SECONDS)
```
or
```java
@Timeout(value = 90, unit = TimeUnit.MILLISECONDS)
```